src/tests/flyer.test.ts

```diff
@@ -276,6 +276,45 @@ describe('getFlyersByCategorySlug tests', () => {
   });
 });
 
+describe('getAllFlyerCategories tests', () => {
+  test('query 3 different flyers with 2 different categories', async () => {
+
+    const randomSlug1 = "Academic";
+    const specificFlyer1 = await FlyerFactory.createSpecific(1, { categorySlug: randomSlug1 });
+    await FlyerModel.insertMany(specificFlyer1);
+
+    const randomSlug2 = "Dance";
+    const specificFlyer2 = await FlyerFactory.createSpecific(1, { categorySlug: randomSlug2 });
+    await FlyerModel.insertMany(specificFlyer2);
+
+    const specificFlyer3 = await FlyerFactory.createSpecific(1, { categorySlug: randomSlug2 });
+    await FlyerModel.insertMany(specificFlyer3);
```

Comment:
A more idiomatic way to write this code would be use an array and leverage `createSpecfic()`'s ability to create more than one of a type of flyer, especially since `insertMany()` allows you to insert multiple documents at once:
```suggestion
    const flyerSlugs = ["Academic", "Dance"];
    let generatedFlyers = [];
    generatedFlyers.concat(await FlyerFactory.createSpecific(1, {categorySlug: flyerSlugs[0]}))
    generatedFlyers.concat(await FlyerFactory.createSpecific(2, {categorySlug: flyerSlugs[1]}))
    
    await FlyerModel.insertMany(generatedFlyers)
```

The code is now a lot shorter/simpler with less variable names to keep track of, while still giving you the same functionality